pkg/controller/deployment/default_cert_manager_creator.go

```diff
@@ -0,0 +1,65 @@
+package deployment
+
+import (
+	"context"
+	"time"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/cert-manager-operator/apis/operator/v1alpha1"
+	alpha1 "github.com/openshift/cert-manager-operator/pkg/operator/clientset/versioned/typed/operator/v1alpha1"
+)
+
+type DefaultCertManagerController struct {
+	operatorClient    v1helpers.OperatorClient
+	controllerFactory *factory.Factory
+	recorder          events.Recorder
+	name              string
+	certManagerClient alpha1.OperatorV1alpha1Interface
+}
+
+func (c *DefaultCertManagerController) Name() string {
+	return c.name
+}
+
+func NewDefaultCertManagerController(controllerName string, operatorClient v1helpers.OperatorClient, certManagerClient alpha1.OperatorV1alpha1Interface, eventsRecorder events.Recorder) factory.Controller {
+	controller := DefaultCertManagerController{
+		operatorClient:    operatorClient,
+		certManagerClient: certManagerClient,
+		controllerFactory: factory.New().ResyncEvery(time.Minute).WithInformers(
+			operatorClient.Informer(),
+		),
+		recorder: eventsRecorder.WithComponentSuffix("default-cert-manager-controller"),
+		name:     controllerName,
+	}
+
+	return controller.controllerFactory.WithSync(controller.sync).ToController("DefaultCertManager_"+controller.name, controller.recorder)
```

Comment:
let's stay consistent with CamelCase

Reply:
Actually, I suggest just to leave `DefaultCertManager`.

Reply:
Fixed.